view/theme/frio/templates/jot-header.tpl

```diff
@@ -166,6 +166,13 @@
 		}
 	}
 
+	function jotGetLanguage() {
+		reply = prompt("{{$whatlang}}", $('#jot-language').val());
+		if(reply && reply.length) {
+			$('#jot-language').val(reply);
```

Comment:
Is this how you plan to add languages to the list?

Reply:
Oh, no. It's a leftover from some tryout where I duplicated the process of the location prompt. Just ignore it. ;-)

Reply:
Or better purge it? I guess this whole PR should be marked as `WIP`?